commands/hltb.js

```diff
@@ -0,0 +1,6 @@
+module.exports.command  = 'hltb'
+module.exports.help     = 'HowLongToBeat'
+module.exports.callback = (message, words) => {
```

Comment:
```suggestion
module.exports.callback = (message) => {
```
Since `words` is not used, should it be removed from the params ?